ArtOfIllusion/src/artofillusion/procedural/Module.java

```diff
@@ -332,17 +338,18 @@ public void getColor(int which, RGBColor color, double blur)
   {
   }
   
-  /** Create a duplicate of this module.  Subclasses with adjustable parameters should
-      override this. */
-  
+  /**
+   * Create a duplicate of this module. Subclasses with adjustable parameters
+   * should override this.
+   */  
   public Module duplicate()
   {
     try
     {
-      Constructor con = getClass().getConstructor(new Class [] {Point.class});
-      return (Module) con.newInstance(new Object [] {new Point(bounds.x, bounds.y)});
+      Constructor con = getClass().getConstructor(Point.class);
+      return (Module) con.newInstance(new Point(bounds.x, bounds.y));
     }
-    catch (Exception ex)
+    catch (IllegalAccessException | IllegalArgumentException | InstantiationException | NoSuchMethodException | SecurityException | InvocationTargetException ex)
```

Comment:
I've already merged, and then had to revert this in master.

How did you get the multi-catch to compile? It's Java7+ and the ant file locks in at Java6

Reply:
Aren't we now using Java 8 as our minimum?

Reply:
I thought that we had decided to wait that until releasing the latest bugfixes (Possibly my misunderstanding)

The build files still reference `source 1.6`, so this should not have compiled...